src/DurableTask.Netherite/Connections/ConnectionInfoExtensions.cs

```diff
@@ -106,98 +105,119 @@ public static Azure.Storage.Blobs.BlobServiceClient GetAzureStorageV12BlobServic
             }
         }
 
-
         /// <summary>
-        /// Creates an Event Hub client for the given connection info.
+        /// Creates an Event Hub Consumer client for the given connection info.
         /// </summary>
         /// <param name="connectionInfo">The connection info.</param>
-        /// <param name="eventHub">The event hub name.</param>
+        /// <param name="eventHubName">The event hub name.</param>
         /// <returns></returns>
-        public static EventHubClient CreateEventHubClient(this ConnectionInfo connectionInfo, string eventHub)
+        public static EventHubConsumerClient CreateEventHubConsumerClient(this ConnectionInfo connectionInfo, string eventHubName)
         {
             if (connectionInfo.ConnectionString != null)
             {
-                var connectionStringBuilder = new EventHubsConnectionStringBuilder(connectionInfo.ConnectionString)
-                {
-                    EntityPath = eventHub
-                };
-                return EventHubClient.CreateFromConnectionString(connectionStringBuilder.ToString());
+                return new EventHubConsumerClient("$Default", connectionInfo.ConnectionString, eventHubName);
```

Comment:
I am not quite sure if this `connectionInfo.ConnectionString` can be used directly to build client without connection string builder. But from the test I did it seems to work.

Reply:
It looks like the constructor takes both a `connectionString` and an `eventHubName`, so it should be fine. The connection string builder was necessary because that constructor expects a connection string for the specific event hub, not the whole namespace.